spec/classes/datadog_agent_ubuntu_spec.rb

```diff
@@ -76,7 +80,10 @@
   end
 end
 
-describe 'datadog_agent::ubuntu::agent6' do
+describe 'datadog_agent::ubuntu' do
+
```

Comment:
We should probably collapse all of these into a single `datadog_agent::ubuntu`  `describe` block with separate contexts for `5`, `6` and `7`, should be easy.